spec-2025x.md.html

```diff
@@ -0,0 +1,1498 @@
+
+Based on the thin-film layer weight $i$, we blend the original Fresnel with the thin-film reflectance for the specular core components
+$$
+OpaqueCore(...) = \mathbf{M}_r (1-i)F(...) + \mathbf{M}_r i R(...)...
+$$
+
+Likewise for transmission
+$$
+TransparentCore(...) = \mathbf{M}_t (1-i)(1-sF(...)) + \mathbf{M}_r i (1-R(...))...
+$$
+
+!!!
+    Please note how the specular weight doesn't affect the thin-film reflectance. It's obvious in the transmission equation above, but also true for the reflection as it uses the actual base IoR instead of the $f_0$ term.
+
+The diffuse BSDF is defined as a blend of the energy correct diffuse BSDF $B_c$ as given in eqn. [b_c] and a new term $B_{c,i}$
+
+\begin{equation}
+\begin{aligned}
+B(\mathbf{v}, \mathbf{l};...) = (1-i)B_c(\mathbf{v}, \mathbf{l}; ...) + i\cdot B_{c,i}(\mathbf{v}, \mathbf{l}; ...)
+\end{aligned}
+\end{equation}
+
+with
+
+\begin{equation}
+\begin{aligned}
+B_{c,i}(\mathbf{v}, \mathbf{l};...) = B_u(\mathbf{v}, \mathbf{l}; ...)( 1 - max(1-E_i(\mathbf{v}\cdot\mathbf{n};...), 1-E_i(\mathbf{l}\cdot\mathbf{n};...))).
+\end{aligned}
+\end{equation}
+
+$E_i$ is the thin-film reflectance adjusted for energy preservation at higher roughness values:
+
+\begin{equation}
+\begin{aligned}
+&E_i(cos\Theta;  r_{uv}, \eta, d_i, \eta_i) = \\
+&\quad\quad\quad lerp\left(lum(R(cos\Theta; r_{uv}, \eta, d_i, \eta_i)), F_{m,avg}(f_{0,R12}, 1), 1 - (1-r_{uv}^2)^5\right)
+\end{aligned}
+\label{weight_scaling}
+\end{equation}
+
+where
+
+- $lum$ is the relative luminance of the colored thin-film reflectance.
+
+- $F_{m,avg}$ is the average Schlick fresnel as defined in eqn.[fr_avg].
+
+- $f_{0,R12}$ the reflectance at normal incidence at the outer thin-film interface.
+
+
+![Figure [thin-film-furnace]: No roughness compensation. Obvious energy loss with increasing roughness.]("img/white_furnace_thin_film[ior_1.5]no_rough_comp.png")
+
+![Figure [thin-film-furnace-comp]: Using the average Schlick Fresnel at higher roughness values mitigates some of the energy loss (eqn. [weight_scaling]).]("img/white_furnace_thin_film[ior_1.5].png")
+
+
+!!! Tip: Diffuse weight approximation
+    Evaluating the full thin-film reflectance twice just for energy correction may be prohibitive. We found that using only the view-dependent term $E_i(\mathbf{v}\cdot\mathbf{n};...)$ and replacing the full reflectance with the $C_0$ term as defined by [#BB17] is a viable approximation.
```

Comment:
The term "full reflectane" was not introduced. Not sure if it's clear to everyone what you are referring to.